app/views/govuk_publishing_components/components/docs/input.yml

```diff
@@ -183,3 +183,17 @@ examples:
         text: "Given name"
       name: given-name
       enterkeyhint: "next"
+  with_dir_attribute:
+    description: |
+      To allow the correct display of right to left languages.
+
+      When `text_direction_rtl` is set to true both the input and the label are displayed right to left.
+
+      When `text_direction_label_ltr` is set to true the textarea is displayed right to left and the label is displayed left to right
```

Comment:
Again this might just be me but I'm finding this a bit confusing. These two options both control the same things, but one overrides the other. I see that you want one flag to set everything to right to left and one flag to override that for the label - can you explain the use case for this? Would it be simpler to have one option for the label and one for the input?

There should also be a test that checks this behaviour as it doesn't look like it behaves as described. If I only set `text_direction_label_ltr` everything is displayed left to right - the textarea is not displayed right to left as described here.